src/translations/index.ts

```diff
@@ -419,4 +432,34 @@ export namespace TranslationsModel {
     export interface ListProjectBuildsOptions extends PaginationOptions {
         branchId?: number;
     }
+
+    export interface PreTranslationReport {
+        languages: TargetLanguage[];
+        preTranslateType: string;
+    }
+
+    export interface TargetLanguage {
+        id: string;
+        files: TargetLanguageFile[];
+        skipped: SkippedInfo;
+        skippedQaCheckCategories: SkippedQaCheckCategories;
+    }
+
+    export interface TargetLanguageFile {
+        id: string;
+        statistics: TargetLanguageFileStatistics;
+    }
+
+    export interface TargetLanguageFileStatistics {
+        phrases: number;
+        words: number;
+    }
+
+    export interface SkippedInfo {
+        [key: string]: any;
+    }
+
+    export interface SkippedQaCheckCategories {
+        [key: string]: any;
+    }
```

Comment:
Instead of `any`, I believe we can use the `CheckCategories` interface exported from the `ProjectsGroups` class